Terraform/wordpress-3tier-app/readme.md

```diff
@@ -0,0 +1,36 @@
+## !! Please this example was tested with aws provider version v.2.13.0!!
+
+### Terraform WordPress deployment
+
+## Description:
+This TF script will deploy a single region highly available WordPress site with RDS, EC2 and VPC into Stratoscale Symphony. 
+
+## Before you begin: Create SSH Key-Pairs
+
+ssh-keygen -f <path to key file> -t rsa -b 2048
+Two files will be created:
+ - no suffix file - the private key
+ - .pub suffix - the public key. This is the file you pass in tfvars
+ 
```

Comment:
Missing the part of how to upload keypair

Reply:
Terraform does it:

resource "aws_key_pair" "app_keypair" {
  public_key = file(var.public_keypair_path)
  key_name   = "wp_app_kp"
}